ouroboros-consensus/src/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -26,6 +31,8 @@ class UpdateLedger blk => ApplyTx blk where
   data family GenTxId blk :: *
 
   -- | Given a 'GenTx', compute its 'GenTxId'.
+  --
+  -- Should be cheap as this will be called often.
```

Comment:
If we're now saying it's cheap, it can and should have a different name from `computeBlah`. The "compute" in the name is a hint that it's not cheap.

Reply:
See #780